### PR TITLE
Circle fix.

### PR DESCRIPTION
**Pull request by jaeilepp**

### CODE REVIEW

**Comment by jaeilepp** [comment]
Seems to work now.

**Comment by agramfort** [comment]
thx heaps @jaeilepp

File: circle.yml [hunk 1]
```diff
@@ -37,7 +37,9 @@ dependencies:
       else
         echo "Conda already set up.";
       fi
-    - conda install -n circleenv --yes numpy scipy scikit-learn mayavi matplotlib sphinx=1.4.1 pillow six IPython pandas;
+    - conda install -n circleenv --yes numpy scipy scikit-learn matplotlib sphinx=1.4.1 pillow six IPython pandas;
+    - conda install -n circleenv --yes traits=4.6.0 traitsui=5.1.0;
+    - conda install -n circleenv --yes mayavi;
```
Comment (Eric89GXL) [inline circle.yml]: Why have these on separate lines? Doing it that way sometimes this breaks conda

**Comment by Eric89GXL** [comment]
Failures on `master` persisted unfortunately :(

**Comment by jaeilepp** [comment]
Yeah, seems so. It's even the same example that succeeded here...

**Pull request merged**